addon/components/o-s-s/copy.ts

```diff
@@ -4,17 +4,31 @@ import { tracked } from '@glimmer/tracking';
 import { inject as service } from '@ember/service';
 
 import ToastService from '@upfluence/oss-components/services/toast';
+import type { IntlService } from 'ember-intl';
 
 interface OSSCopyArgs {
   value: string;
   inline?: boolean;
 }
 
 export default class OSSCopy extends Component<OSSCopyArgs> {
-  @service intl: any;
+  @service declare intl: IntlService;
   @service declare toast: ToastService;
+
+  @tracked accessibleClipboard: boolean = false;
   @tracked inline: boolean = this.args.inline ?? false;
 
+  constructor(owner: unknown, args: OSSCopyArgs) {
+    super(owner, args);
+
+    navigator.permissions
+      .query({ name: 'clipboard-write' as PermissionName })
+      .then(({ state }) => {
+        this.accessibleClipboard = state === 'granted';
+      })
+      .catch(() => {});
```

Comment:
any reason to declare catch if we're not using it for anything ?

Reply:
it's so that if the feature detection somehow fail, we consider that it's not accessible but don't want to send it to Sentry either. Or should we? 🤔

Reply:
here are the exceptions that could be thrown https://developer.mozilla.org/en-US/docs/Web/API/Permissions/query#exceptions